packages/python/tests/test_normalization.py

```diff
@@ -0,0 +1,137 @@
+import sys
+import os
+
+# Add the parent directory to the path for imports
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
+
+from genai_prices.types import find_provider_by_id, Provider, ClauseOr, ClauseEquals, ClauseContains
+
+# Mock providers for testing
+mock_providers = [
```

Comment:
probably better to test against actual providers, not create them just for testing.

Reply:
absolutely, updated this part.